shell/renderer/api/atom_api_web_frame.cc

```
@@ -314,6 +314,16 @@ void SetSpellCheckProvider(mate::Arguments* args,
     return;
   }
 
+  std::vector<std::string> language_vector;
+  bool is_multi_lang_spell_check = languages->IsArray();
+  if (is_multi_lang_spell_check) {
+    gin::ConvertFromV8(args->isolate(), languages, &language_vector);
+  } else {
+    std::string language;
+    gin::ConvertFromV8(args->isolate(), languages, &language);
```

Same with this conversion, if both fail we should throw an error